src/onDataTransform.js

```diff
@@ -1,7 +1 @@
-export default function onDatatransform() {
```

Comment:
any specific reason to move it to preprocess? Or just trying to keep everything together?

Reply:
Because it's a pain to modify `current_data`, particularly when there's more than one mark type.  Simpler to modify the raw data, hence `initial_data`, `measure_data`, etc.